local-postgres.md

```diff
@@ -0,0 +1,99 @@
+# Guide for using a local postgres database with QAE and core-grpc-postgres-connector
+
+## First step: Setting up the example
+
+Verify that it´s possible to spin up the docker environment and run the example as described in the [README.md](./README.md).
+
+After running all the steps a sample of the airports table should be visible in the console.
+
+## Second step: Dump the airports table
+
+This section can be skipped and the [airports.tar](./example/postgres-image/airports.tar) be used in the next step
```

Comment:
The `tar` file should be added to the `.gitattributes` as a binary if we want to check it in. Perhaps better to just document the command for creating a tar file from the csv?

Reply:
It´s not possible to create directly from the csv. Added it to .gitignore